packages/ndla-ui/src/ResourceBox/ResourceBox.tsx

```diff
@@ -0,0 +1,193 @@
+/**
+ * Copyright (c) 2022-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+import React, { ReactNode } from 'react';
+import { breakpoints, fonts, mq } from '@ndla/core';
+import Button, { StyledButton } from '@ndla/button';
+import BEMHelper from 'react-bem-helper';
+import { useTranslation, WithTranslation } from 'react-i18next';
+import { Launch } from '@ndla/icons/common';
+import { LicenseByline } from '@ndla/licenses';
+import { messagesNB } from '@ndla/ui';
```

Comment:
Importer aldri messagesNB direkte! Bruk berre key for korrekt verdi. Sjå kommentar lenger nede.